tests/fixtures/testModel.ts

```diff
@@ -154,6 +154,8 @@ export interface TestModel extends Model {
     maybeString: Maybe<string>;
     justNull: null;
   };
+
+  stringAndBoolArray?: Array<string | boolean>;
```

Comment:
This should pass without your change. 

```suggestion
  stringAndBoolArray?: Array<(string | boolean)>;
```

Reply:
The original format was `(string | boolean)[]`, but eslint changed this. I fixed this like you suggested but need to add prettier-ignore and move up in object structure because prettier had some weird bug and it added double semicolon to end of the line when ignored.